src/frontend/qt_sdl/main.cpp

```diff
@@ -3335,6 +3356,12 @@ int main(int argc, char** argv)
     Input::OpenJoystick();
 
     mainWindow = new MainWindow();
+    if (options->fullscreen)
+    {
+        // onFullscreenToggled is private and I don't know if I should copy what's inside it or make it public
```

Comment:
add a public function something like setFullscreen and put the logic from onFullscreenToggled there so that it's not duplicated.